plugindevelopment/webhookplugins.md

```diff
@@ -89,6 +89,34 @@ For detailed information about the hooks and when they are called, see the secti
 Note, when overriding a **mountpoint** or a **client-id** both have to be returned by the webhook implementation for it to have an effect.
 {% endhint %}
 
+### Responses
+
+All hooks, unless stated otherwise, respond with a JSON-encoded payload and a success code of 200. All hooks support responding with "ok", indicated that the request was successful. 
+
+```javascript
+{
+    "result": "ok"
+}
+```
+
+Other possible responses are "next", meaning that the next callback should be tried. 
+
+```javascript
+{
+  "result": "next"
+}
+```
+
+Errors, e.g. authentication failures, are returned by a an "error" payload, either with the predefined "not_allowed" or some other error text:
+
+```javascript
+{
+  "result": {
+    "error": "not_allowed"
+  }
+}
+```
+
```

Comment:
Can we keep
{
  "result": {
    "error": "some_error_message"
  }
}
as a literal example.